trieste/space.py

```diff
@@ -411,25 +432,49 @@ def __repr__(self) -> str:
                 tags = {self.subspace_tags})
                 """
 
+    @property
+    def lower(self) -> TensorType:
+        """The lowest values taken by each space dimension, concatenated across subspaces."""
+        lower_for_each_subspace = [self.get_subspace(tag).lower for tag in self.subspace_tags]
+        return tf.concat(lower_for_each_subspace, -1)
```

Comment:
(we usually write it like this, no?)

```suggestion
        return tf.concat(lower_for_each_subspace, axis=-1)
```